internal/testutils/olm.go

```diff
@@ -81,11 +81,30 @@ func (tc TestContext) AddPackagemanifestsTarget() error {
 	return nil
 }
 
-// DisableOLMBundleInterativeMode will update the Makefile to disable the interactive mode
-func (tc TestContext) DisableOLMBundleInteractiveMode() error {
+// disableOLMBundleInteractiveMode will update the Makefile to disable the interactive mode
+func (tc TestContext) disableOLMBundleInteractiveMode() error {
 	// Todo: check if we cannot improve it since the replace/content will exists in the
 	// pkgmanifest target if it be scaffolded before this call
 	content := "operator-sdk generate kustomize manifests"
 	replace := content + " --interactive=false"
 	return ReplaceInFile(filepath.Join(tc.Dir, "Makefile"), content, replace)
 }
+
+// IntegrateProjectWithOLM runs all commands to integrate the project with OLM
+func (tc TestContext) IntegrateProjectWithOLM() error {
```

Comment:
Lets call this function by what it's really doing
```suggestion
func (tc TestContext) CreateBundleImage() error {
```

Reply:
Done